data/language/en-GB.txt

```diff
@@ -730,10 +730,10 @@ STR_1339    :{BLACK}No test results yet…
 STR_1340    :{WINDOW_COLOUR_2}Max. speed: {BLACK}{VELOCITY}
 STR_1341    :{WINDOW_COLOUR_2}Ride time: {BLACK}{STRINGID}{STRINGID}{STRINGID}{STRINGID}
 STR_1342    :{DURATION}
-STR_1343    :{DURATION} / 
+STR_1343    :{DURATION} /
```

Comment:
You probably have to revert all of these whitespace changes on this file.

Reply:
Whoops, noticed it when I originally had to do it but then forgot it when I had to rebase and change the string number.